addons/core/XEH_PREP.sqf

```diff
@@ -65,6 +65,8 @@ PREP(getDefaultRadioClasses);
 PREP(getRadioOwner);
 PREP(getSwSpeakers);
 PREP(getLrSpeakers);
+PREP(getSwHalfDuplexOverride);
```

Comment:
New wording is SR(Short-Range) instead of SW (short-wave) and LR (Long-range)
Old function names were kept as Sw for backwards compat. But everything Internal that is not exposed as part of the API is using SR already. 
Might be confusing for people though that some things are Sw and some Sr.
This change was meant to make it easier for Players. Modders can probably cope with Sw meaning short-range.
Your choice.